schemas/rdf/rdf-ontology.ttl

```diff
@@ -107,10 +107,10 @@ aas:AccessControlPolicyPoints rdf:type owl:Class ;
     rdfs:range aas:PolicyDecisionPoint ;
 .
 
-###  https://admin-shell.io/aas/3/0/RC01/AccessControlPolicyPoints/policyEnforcementPoint
-<https://admin-shell.io/aas/3/0/RC01/AccessControlPolicyPoints/policyEnforcementPoint> rdf:type owl:ObjectProperty ;
-    rdfs:comment "The access control policy enforcement point of the AAS."@en ;
-    rdfs:label "has policy enforcement point"^^xsd:string ;
+###  https://admin-shell.io/aas/3/0/RC01/AccessControlPolicyPoints/policyEnforcementPoints
+<https://admin-shell.io/aas/3/0/RC01/AccessControlPolicyPoints/policyEnforcementPoints> rdf:type owl:ObjectProperty ;
+    rdfs:comment "The access control policy enforcement points of the AAS."@en ;
```

Comment:
policyEnforcementPoint in (https://admin-shell.io/aas/3/0/RC01/AccessControlPolicyPoints/policyEnforcementPoint) the property path should be in a single form.